__tests__/components/__snapshots__/dashtable_v2.test.jsx.snap

```diff
@@ -59,7 +59,7 @@ exports[`GlobalDashboardTable renders dashtable with correct data format 1`] = `
           <div
             class="bg-red-600 h-5 w-5"
             style="margin: auto;"
-            title="Last completion time: 9/17/2018, 6:01:46 PM"
+            title="Last completion time: 9/17/2018, 11:01:46 AM"
```

Comment:
8 hour time difference and NNBSP used in snapshot.
output on workflow uses regular white space character ' '